Tone Finder

fponce004 opened the pull request:
Added codes to search for resonator tones and example python notebook.

fponce004 committed:
Tone Finder

Added codes to search for resonator tones and example python notebook.

nmaterise commented on pna_control/tone_finder.py:
```diff
@@ -0,0 +1,191 @@
+def loadVNA(FileList):
+    """
+    The data structure anticipated is single line header of columns with columns units in freq [Ghz], Power [dB], Phase [rad].
+    The data phase will be corrected without request if the phase exceeds 10 'degree' into radians.  
```
Can you do the same if the frequency data is in Hz and not GHz? Our frequency data is usually in Hz and phase data is in degrees.

nmaterise reviewed (commented):
From what I can review in tone_finder.py, everything looks good besides my one comment. I was unable to read the ipynb.

fponce004 committed:
Updated Tone_finder.py with new features

Updated loadVNA() to change Hz into GHz by default. In __name__ removed path specific to my computer to generalized statement. Added new function electronicDelayCorrection() which is not integrated into other code. This allows user to correct for the electronic delay from the physical setup without applying the unwrap function to the phase data.
